inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/reduce_gpu_b_fs_yx_fsv16.cl

```diff
@@ -341,17 +341,7 @@ uint offset = batch_out * input_batch_pitch + ((feature_out + FSV - 1) / FSV) *
         for (uint fi = feature_out; fi < feature_max_val; fi += FSV) {
             for (uint yi = y_out; yi < y_max_val; ++yi) {
                 for (uint xi = x_out; xi < x_max_val; ++xi) {
-                    INPUT_VEC input = (INPUT_VEC)(INPUT_INIT_VAL);
-                    #if REDUCE_FEATURE && (INPUT0_FEATURE_NUM % FSV != 0)
-                        if (fi + FSV <= INPUT0_FEATURE_NUM)
-                            input = BLOCK_READ(data, offset);
-                        else
-                            if (fi + get_sub_group_local_id() < INPUT0_FEATURE_NUM)
-                                for (int i = 0; i < READ_OFFSET; ++i)
-                                    input[i] = data[offset + get_sub_group_local_id() + i * get_max_sub_group_size()];
-                    #else
-                        input = BLOCK_READ(data, offset);
-                    #endif
+                    INPUT_VEC input = BLOCK_READ(data, offset);
```

Comment:
As I understand, you can simplify this code due to rounding gws dims to simd for both branches in .cpp file?

Reply:
No, I did simplify it because remainder of f-axis could read with BLOCK_READ even if rest garbage value after remainder is read, and then the garbage value never be used.